### DISCUSSION TITLE
[Projects Beta] - Convert draft issue to issue in org repository

### DISCUSSION THREAD

**Discussion by JoshBeveridge** [community/community#12350] [Projects and Issues]
# Context
- working from a private Beta Project created on my personal account (e.g. profile / projects / projects beta)
- I am able to add issues from repositories I own
- I am able to create unlinked draft issues
- I can currently copy an issue/PR's link from anywhere on Github and add it to this project

# Problem
- when I try to convert a draft issue to an issue, the search that appears only shows repositories I own
- I need to be able to convert my draft issues to issues on repositories in organizations I'm a part of for effective project management

I'm unsure if this is something I'm missing (permissions? visibility? org administration?) or something that just isn't currently possible. Either way, any help/feedback would be super appreciated, and I hope we can see this functionality soon if it's not already possible.

**Comment by lina-kim** [comment]
Similar suggestion here, though my context is a fork of an organization's repository. I'm a member of the organization (though not an admin), and I'm unable to convert draft issues to issues on my fork. It would be so helpful to use Projects to organize issues on my fork, instead of using manual workarounds.

**Comment by jchouse** [comment]
Had the same problem, i resolved it when enabling Issue for my repository in the settings. Because having Projects in the repository doesn't enable Issue by default.

**Comment by lina-kim** [comment]
Thanks for the reminder this exists @jchouse! I'd tried your solution six months ago to no avail, but GitHub must've made some updates - I can now convert drafts > issues on my fork though I haven't changed anything since then. 

I'm not currently an owner of any organizations, so @JoshBeveridge will have to give this a whirl to determine whether his original problem is resolved.

**Comment by JoshBeveridge** [comment]
Thanks for the nudge @lina-kim - a quick test suggests I'm still having the same problem. My private project board still can't see my org level repositories despite being an owner. 
![image](https://user-images.githubusercontent.com/6960386/188175515-c6d5e259-3205-4eab-bb11-15e3a439f86d.png)

**Comment by morficus** [comment]
I came across this thread two years later while trying to deal with the same issue (Project is in Org A, repo is in Org B)

Looks like we are still not able to convert draft issues into a real issue cross-org, but I did find Project update blog from 2023 that talked about a possible workaround: https://github.blog/changelog/2023-02-23-github-issues-projects-february-23rd-update/#add-cross-organization-issues-and-pull-requests-to-projects

From the Project board, using the syntax `#org-b/repo-name` will create a new issue in any project across any org (as long as you have access). Unfortunately this syntax does not work for converting drafts into issues (or at least not yet)

**Comment by italo0072** [comment]
good
